### PR TITLE
DOC: add some what's new with respect to #13888

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: docs/source/whatsnew/version8.rst [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+
+The biggest improvement for 8.9 is a drastic amelioration if the
+auto-suggestions sponsored by D.E. Shaw and implmented by the more and more
+active contributor `@krassowski <https://github.com/krassowski>`.
+
+- ``right`` accepts a single character from suggestion
+- ``ctrl+right`` accepts a semantic token (macos default shortcuts take
+  precedence and need to be disabled to make this work)
+- ``backspace`` deletes a character and resumes hinting autosuggestions
+- ``left`` accepts suggestion and moves cursor left one character.
```
Comment (krassowski) [inline docs/source/whatsnew/version8.rst:19]: Was changed to ctrl left

File: docs/source/whatsnew/version8.rst [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+- ``down`` moves to suggestion to later in history when no lines are present below the cursors. 
+- ``up`` moves to suggestion from earlier in history when no lines are present above the cursor.
+
+This is best described by the Gif posted by @krassowski, and in the PR itself
+:ghpull:`13888`.
+
+.. image:: ../_images/autosuggest.png
+
+Please report any feedback in order for us to improve the user experience. 
+In particular we are also working on make the shortcuts configurable.
```
Comment (krassowski) [inline docs/source/whatsnew/version8.rst:30]:
```suggestion
In particular we are also working on making the shortcuts configurable.
```

File: docs/source/whatsnew/version8.rst (hunk 1 above)
Comment (Carreau) [inline docs/source/whatsnew/version8.rst:19]:
```suggestion
- ``ctrl-left`` accepts suggestion and moves cursor left one character.
```

File: docs/source/whatsnew/version8.rst [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+- ``backspace`` deletes a character and resumes hinting autosuggestions
+- ``ctrl-left`` accepts suggestion and moves cursor left one character.
+- ``backspace`` deletes a character and resumes hinting autosuggestions
+- ``down`` moves to suggestion to later in history when no lines are present below the cursors. 
+- ``up`` moves to suggestion from earlier in history when no lines are present above the cursor.
+
+This is best described by the Gif posted by @krassowski, and in the PR itself
+:ghpull:`13888`.
+
+.. image:: ../_images/autosuggest.png
```
Comment (Carreau) [inline docs/source/whatsnew/version8.rst:27]:
```suggestion
.. image:: ../_images/autosuggest.gif
```

**Pull request merged**